packages/app-portal/src/systems/Bridge/pages/Bridge.tsx

```diff
@@ -123,8 +123,9 @@ export const Bridge = () => {
           to withdraw back to Ethereum. Learn more about our architecture and
           security in our&nbsp;
           <Link
-            isExternal
+            target="_blank"
```

Comment:
You missed the `isExternal` prop (that property is adding an icon there)

```suggestion
            isExternal
            target="_blank"
```

Reply:
Oh, my bad. Now Done.